ReactAndroid/src/main/java/com/facebook/react/views/art/ARTSurfaceViewShadowNode.java

```diff
@@ -33,6 +35,18 @@
 
   private @Nullable Surface mSurface;
 
+  private boolean mIsBackgroundColorSet;
+  private int mBackgroundColor;
```

Comment:
We could make this `@Nullable Integer` to avoid having to sync `mIsBackgroundColorSet`